### PR TITLE
gxemul: update to version 0.7.0

### PR DESCRIPTION
**Pull request by stechert**
* update to version 0.7.0
* update the location of the homepage to the current homepage
* added modeline

Closes: https://trac.macports.org/ticket/66915

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

1. macOS 13.4 22F66 arm64 Xcode 14.3.1 14E300c
2. macOS 13.4 22F66 x86_64 Xcode 14.3 14E222b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: emulators/gxemul/Portfile [hunk 1]
```diff
@@ -13,16 +14,20 @@ consisting of processors (ARM, MIPS, Motorola 88K, PowerPC, and SuperH) \
 and surrounding hardware components such as framebuffers, interrupt \
 controllers, busses, disk controllers, and serial controllers.
 
-homepage        http://gxemul.sourceforge.net/
-master_sites    ${homepage}src/
+# There is also a sourceforge hosted subversion repo at https://sourceforge.net/projects/gxemul/
+homepage        https://gavare.se/gxemul/
+master_sites    https://gavare.se/gxemul/src/
```
Comment (herbygillot) [inline emulators/gxemul/Portfile:19]: You can actually do `${homepage}/src/` here

**Comment by herbygillot** [comment]
> I'll make the changes and resubmit.

Thank you. Please amend the same commit you have in the PR.

File: emulators/gxemul/Portfile (hunk 1 above)
Comment (ryandesign) [reply emulators/gxemul/Portfile:19]: To avoid a double slash in the download URL, you would want it to be `${homepage}src/`.

**Review by reneeotten — CHANGES_REQUESTED** [review]
the `license` field is missing; additionally, it doesn't look like the other requested changes are made and there is now a "merge" commit which we don't want. Please make the changes, rebase with master, and then force-push to this PR.

**Comment by reneeotten** [comment]
ping @stechert

**Comment by reneeotten** [comment]
feel free to reopen once you made the changes

**Pull request closed without merging**